### ISSUE TITLE
image for window is in the rancher-images.txt for linux

### ISSUE THREAD

**Issue by jiaqiluo** [rancher/rancher#22463]
**What kind of request is this (question/bug/enhancement/feature request):**
Bug

**Steps to reproduce (least amount of steps as possible):**
- run Rancher:master-head 0f08172
- go to `v3/kontainerdrivers/rancher-images` to download the file `rancher-images.txt`

**Result:**
- the image `rancher/kubelet-pause:v0.1.3` is in the list, but it is not for Linux platform
```
root@localhost:~# docker pull rancher/kubelet-pause:v0.1.3
v0.1.3: Pulling from rancher/kubelet-pause
no matching manifest for linux/amd64 in the manifest list entries
```

We should not include this image if Rancher is running in a Linux environment.

**Comment by jiaqiluo** [comment]
The bug fix is validated in rancher:master-head a98b70f17

Now the image `rancher/kubelet-pause:v0.1.3` is not in the downloaded `rancher-images.txt` file

**Comment by jiaqiluo** [comment]
Reopen the issue because this image  is shown in the rancher-image.txt file downloaded from Github release page for v2.3.0-rc1 
https://github.com/rancher/rancher/releases/tag/v2.3.0-rc1

**Comment by kinarashah** [comment]
This is getting fixed by https://github.com/rancher/rancher/pull/22443

**Comment by jiaqiluo** [comment]
in v2.2.3-rc2

```
Error response from daemon: No such image: rancher/kubelet-pause:v0.1.3
Error response from daemon: No such image: rancher/wmi_exporter-package:v0.0.2
```

**Comment by loganhz** [comment]
We are using manifest images for Windows.

`rancher/hyperkube:v1.15.3-rancher3` is a manifest 

In a Windows cluster, both linux node and windows node will pull it from the private registry. If we don't support manifest, we cannot do the air-gap install for Windows cluster.

I think we should delete `rancher-windows-images.txt`,  put all images into `rancher-images.txt` and support manifest images in `rancher-save-images.sh` and `rancher-load-images.sh`.

cc @cloudnautique @drpebcak

**Comment by loganhz** [comment]
Can be validated in next RC

**Comment by jiaqiluo** [comment]
The bug fix is validated in v2.3.0-rc5 that there are no Windows images in the `rancher-images.txt` file
